pkgs/by-name/un/unl0kr/package.nix

```diff
@@ -1,30 +1,33 @@
-{ lib
-, nixosTests
-, stdenv
-, fetchFromGitLab
-, inih
-, libdrm
-, libinput
-, libxkbcommon
-, meson
-, ninja
-, pkg-config
-, scdoc
+{
+  lib,
+  nixosTests,
+  stdenv,
+  fetchFromGitLab,
+  inih,
+  libdrm,
+  libinput,
+  libxkbcommon,
+  meson,
+  ninja,
+  pkg-config,
+  scdoc,
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "unl0kr";
```

Comment:
If the project changed names to buffybox, don't we need to change all the names in nixpkgs too? Including the NixOS options?

Reply:
Buffybox is the suite, unl0kr is unl0kr still. Maybe buffybox-unl0kr? Although at that point might as well package bufyboard